Neos.Neos/Resources/Private/Templates/Module/Administration/Users/Index.html

```diff
@@ -3,75 +3,87 @@
 
 <f:section name="content">
 	<div class="neos-row-fluid">
+		<f:form action="index">
```

Comment:
Wouldn't it be nice to display some message if no user can be found?

Reply:
Done